index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset='utf-8'>
+  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+  <title>Page Title</title>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+  <script src='main.js'></script>
```

Comment:
Please fix this broken link! Make sure you have the javaScript file present on your repository! Placing this file at this level is not a good practice because scripts, historically, prevented additional resources from being downloaded faster. By placing them at the bottom, your style, content, and media could be uploaded faster, giving the perception of improved performance.